src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
```

Comment:
The variable name 'KEYS' is misleading. It actually contains the values of the 'robot' object. Consider renaming it to 'VALUES' or something similar that accurately represents its contents.